### ISSUE TITLE
Is it possible if I can see how the validation accuracy changes over the number of epochs for DeiT?

### ISSUE THREAD

**Issue by CharlesLeeeee** [facebookresearch/deit#193]
Hi, is it possible if I can see how the validation accuracy changes over the number of epochs? I would like to see if my training process is going in the right direction compared to you guys when training DeiT on ImageNet?
